cockatrice/src/setsmodel.h

```diff
@@ -43,7 +43,8 @@ class SetsModel : public QAbstractTableModel {
     QStringList mimeTypes() const;
     void swapRows(int oldRow, int newRow);
     void toggleRow(int row, bool enable);
-    void toggleAll(bool enable);
+    void toggleRow(int row);
+    void toggleAll(int rowToCopyStatusFrom);
```

Comment:
I still think it's weird behavior. Toggle all should do

if any row enabled:
  disable all
else:
  enable all

But as we talked about it can happen in another pr if you want. Now that the release is out though I don't see the rush, so it probably makes sense to just do it now